lesson-4/script.js

```diff
@@ -0,0 +1,137 @@
+  // Вивести на сторінку в один рядок через кому числа від 10 до 20.
+
```

Comment:
1. Why do you have indent spaces all over the file?
```suggestion
// Вивести на сторінку в один рядок через кому числа від 10 до 20.

```